tests/v2/validation/scaling/scaling_nodepools.go

```diff
@@ -0,0 +1,71 @@
+package scaling
+
+import (
+	"testing"
+
+	apisV1 "github.com/rancher/rancher/pkg/apis/provisioning.cattle.io/v1"
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	v1 "github.com/rancher/rancher/tests/framework/clients/rancher/v1"
+	"github.com/rancher/rancher/tests/framework/extensions/machinepools"
+	"github.com/rancher/rancher/tests/framework/extensions/provisioninginput"
+	rke1 "github.com/rancher/rancher/tests/framework/extensions/rke1/nodepools"
+	"github.com/rancher/rancher/tests/framework/extensions/scaling"
+	"github.com/rancher/rancher/tests/framework/extensions/workloads/pods"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	ProvisioningSteveResourceType = "provisioning.cattle.io.cluster"
+	defaultNamespace              = "fleet-default/"
```

Comment:
Nit; I think we should append this ` /` in between ID calls

Reply:
I can do that